variables.tf

```diff
@@ -225,53 +225,50 @@ variable "gcp_sa_private_key" {
   default     = null
 }
 
-variable "gcp_sa_project_id" {
+# GCP Storage Configuration
+variable "gcp_artifact_registry_url" {
   type        = string
-  description = "GCP Project ID associated with the Service Account"
+  description = "URL for GCP Artifact Registry (e.g., 'LOCATION-docker.pkg.dev/PROJECT_ID/REPOSITORY')"
```

Comment:
Will the registry url contain the repository?

Reply:
no, you're right, `LOCATION-docker.pkg.dev/PROJECT_ID` 
sent the fix.